scipy/spatial/setup.py

```diff
@@ -105,7 +106,15 @@ def configuration(parent_package='', top_path=None):
                                **numpy_nodepr_api)
     ext._pre_build_hook = pre_build_hook
 
-    config.add_extension('_voronoi',
+    if int(os.environ.get('SCIPY_USE_PYTHRAN', 1)):
+        import pythran
+        ext = pythran.dist.PythranExtension(
+            'scipy.spatial._voronoi',
+            sources=["scipy/spatial/_voronoi.py"],
```

Comment:
the build/runtime system will prioritize the `_voronoi` extension over the Python module with the same name? I might have left the source file with the same pythran-specific name but built the extension to shadow Cython name for example, if that's possible.